qscintilla2/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+BIN=$PREFIX/bin
+
+if [ `uname` == Darwin ]; then
+	## Build main dylib ##
+
+	# Go to Qscintilla source dir and then to its Qt4Qt5 folder.
+	cd ${SRC_DIR}/Qt4Qt5
+	# Build the makefile with qmake, specify llvm as the compiler
+	# The normal g++ compiler causes an __Unwind_Resume error at linking phase
+	${BIN}/qmake qscintilla.pro -spec macx-llvm
+	# Build Qscintilla
+	make
+	# and install it
+	make install
+
+	## Build Python module ##
+
+	# Go to python folder
+	cd ${SRC_DIR}/Python
+	# Configure compilation of Python Qsci module
+	${PYTHON} configure.py --qmake=${BIN}/qmake --sip=${BIN}/sip --qsci-incdir=${PREFIX}/include --qsci-libdir=${PREFIX}/lib --spec=macx-llvm --no-qsci-api 
```

Comment:
Alright, let me know what I can do from here. I also successfully managed to compile Qscintilla2 against (Py)Qt5 with a minimum of changes to this script, but this was for Homebrew Python/Qt. It shouldn't be that much of a stretch for the Anaconda environment. Good to hear Anaconda is also pushing towards Qt5! Once it does so officially, I can see if I can adapt these build scripts accordingly